### PR TITLE
Fix sles12 image

### PR DESCRIPTION
**Pull request by yinzhi7**
1. sles12 image moved to new repo
2. automake is no longer in that folder

### CODE REVIEW

**Comment by yinzhi7** [comment]
Already tested on pantheon cloud build trigger job

File: build/x86_64_sles12/Dockerfile [hunk 1]
```diff
@@ -28,9 +28,9 @@ RUN zypper addrepo https://download.opensuse.org/repositories/devel:libraries:c_
         varnish-devel \
         wget \
         which \
- && wget ftp://ftp.pbone.net/mirror/ftp5.gwdg.de/pub/opensuse/repositories/home:/Herbster0815/openSUSE_Factory/noarch/automake-1.14.1-56.1.noarch.rpm \
- && rpm --force -Uvh automake-1.14.1-56.1.noarch.rpm \
- && rm automake-1.14.1-56.1.noarch.rpm \
- && zypper -n clean \
+ && wget https://ftp.gwdg.de/pub/opensuse/repositories/home%3A/Herbster0815/openSUSE_Leap_15.1/noarch/automake-1.16.1-lp151.92.1.noarch.rpm \
+ && rpm --force -Uvh automake-1.16.1-lp151.92.1.noarch.rpm \
+ && rm automake-1.16.1-lp151.92.1.noarch.rpm \
```
Comment (igorpeshansky) [inline build/x86_64_sles12/Dockerfile]: Careful — I think our current build requires automake 1.14 (automake versions aren't backward-compatible). Did you test the build steps in the new container?

**Comment by yinzhi7** [comment]
Good point. I was having a hard time to find the 1.14.1  in the opensuse
repo; therefore, I used 1.16.1. I did find a 1.41.1 rpm but not inside that
repo. At step of rpm automaker complaining that one is not a rpm pkg. so I
went back to the original repo and used 1.16.1.
I only tested it successfully for building the container image. I will let
you know how it goes when I try building the pkg in the build container.
Cheers,

File: build/x86_64_sles12/Dockerfile [hunk 2]
```diff
@@ -28,9 +28,6 @@ RUN zypper addrepo https://download.opensuse.org/repositories/devel:libraries:c_
         varnish-devel \
         wget \
         which \
- && wget ftp://ftp.pbone.net/mirror/ftp5.gwdg.de/pub/opensuse/repositories/home:/Herbster0815/openSUSE_Factory/noarch/automake-1.14.1-56.1.noarch.rpm \
- && rpm --force -Uvh automake-1.14.1-56.1.noarch.rpm \
- && rm automake-1.14.1-56.1.noarch.rpm \
- && zypper -n clean \
  && /bin/sed -i -e 's/VERSION = 42.3/VERSION = 12/' /etc/SuSE-release \
```
Comment (igorpeshansky) [inline build/x86_64_sles12/Dockerfile]: Let's add the same comment here as in the logging agent repo: `# Pretend we're on SLES 12.`.

File: build/x86_64_sles12/Dockerfile (hunk 1 above)
Comment (igorpeshansky) [reply build/x86_64_sles12/Dockerfile]: Actually, our current implementation requires automake 1.14 or later. The one that comes with OpenSUSE Leap 42.3 is apparently too old.

File: build/x86_64_sles12/Dockerfile [hunk 3]
```diff
@@ -28,9 +28,10 @@ RUN zypper addrepo https://download.opensuse.org/repositories/devel:libraries:c_
         varnish-devel \
         wget \
         which \
- && wget ftp://ftp.pbone.net/mirror/ftp5.gwdg.de/pub/opensuse/repositories/home:/Herbster0815/openSUSE_Factory/noarch/automake-1.14.1-56.1.noarch.rpm \
- && rpm --force -Uvh automake-1.14.1-56.1.noarch.rpm \
- && rm automake-1.14.1-56.1.noarch.rpm \
- && zypper -n clean \
+ && wget https://ftp.gwdg.de/pub/opensuse/repositories/home%3A/Herbster0815/openSUSE_Leap_15.1/noarch/automake-1.16.1-lp151.92.1.noarch.rpm \
```
Comment (igorpeshansky) [inline build/x86_64_sles12/Dockerfile]: I'm a little worried about installing a package from Leap 15.1 on Leap 42.3. Can we not find a build for Leap 42.3?

Comment (igorpeshansky) [reply build/x86_64_sles12/Dockerfile]: This seems to work, though, so let's keep it.

**Pull request merged**